MODULE.bazel

```diff
@@ -45,6 +45,13 @@ single_version_override(
     version = "6.0",
 )
 
+archive_override(
+    module_name = "rules_java",
+    urls = ["https://github.com/bazelbuild/rules_java/archive/0355f81fb53a9ca267d69c0170d53a7b3f854d9d.tar.gz"],
```

Comment:
Looks like this hasn't been updated after you synced your branch?
I see your latest commit is:
https://github.com/bazelbuild/rules_java/commit/13e85b4ad6c142aea39605593e98c8d971981369

Reply:
@iancha1992 (same for workspace_deps.bzl below)

Reply:
Sorry, forgot about that! Thank you!